gk/main.c

```diff
@@ -2220,6 +2220,11 @@ get_responsible_gk_mailbox(const struct ip_flow *flow,
 	uint16_t queue_id;
 	int block_idx;
 
+	if (!gk_conf->net->front.rss) {
```

Comment:
1. Let's use `unlikely()` in the test. A Gatekeeper/Grantor server **in production** without RSS is not likely.

2. I can't comment on the line `uint32_t rss_hash_val = rss_ip_flow_hf(flow, 0, 0);` above, but move the call to `rss_ip_flow_hf()` after this `if` statement.